news/utils-updates.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* Updated functions related to changes in `diffpy.utils` version 3.6.0.
```

Comment:
Can you try and think more precisely about how to say this.  "Updated" is the same as "changed" so the message contains close to zero information.....